packages/eslint-plugin-qwik/tests/valid-lexical-scope/valid-no-serialize-symbol.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  useTask$,
+  component$,
+  NoSerializeSymbol,
+  SerializerSymbol,
+  useSerializer$,
+} from '@qwik.dev/core';
+
+class NoSerSym {
+  [NoSerializeSymbol] = true;
+}
+
+class SerSym {
+  [SerializerSymbol]() {
+    return this.toString();
+  }
+}
+
+export interface Value {
+  value: number;
+  obj1: NoSerSym;
+  obj2: SerSym;
+}
+
+export const HelloWorld = component$(() => {
+  const state: Value = { value: 12, obj1: new NoSerSym(), obj2: new SerSym() };
```

Comment:
I am a bit confused as to what happens here:
- What will the `obj1` be after resuming? Presumably `undefined` 
- What will the `obj2` be after resuming? Not sure how the deserializer will restore the value.